doc/1/controllers/index/create/index.md

```diff
@@ -0,0 +1,24 @@
+---
+code: true
+type: page
+title: create
+description: Creates an index
+---
+
+# create
+
+Creates a new index in Kuzzle
+
+## Arguments
+
+```cs
+Task CreateAsync(string index);
+```
+
+| Arguments | Type                       | Description       | Required |
```

Comment:
I don't think the field `Required` is relevant since you're forced to pass an `string index` to the method.

Same thing every other methods.